packages/grpc-js/src/load-balancer-eds.ts

```diff
@@ -297,16 +297,33 @@ export class EdsLoadBalancer implements LoadBalancer {
         WeightedTarget
       >();
       for (const localityObj of localityArray) {
+        /* Use the endpoint picking policy from the config, default to
+         * round_robin. */
+        const endpointPickingPolicy: LoadBalancingConfig[] = [
+          ...this.lastestConfig.eds.endpointPickingPolicy,
+          { name: 'round_robin', round_robin: {} },
+        ];
+        let childPolicy: LoadBalancingConfig[];
+        if (this.lastestConfig.eds.lrsLoadReportingServerName) {
+          childPolicy = [
+            {
```

Comment:
So I suppose this is an object being loaded into the `childPolicy` array, which is of `LoadBalancingConfig[]` type, but I suppose the `LoadBalancingConfig` type doesn't have a generics class, so the object itself can have arbitrary key names like `lrs` below in L311, or the `round_robin` key up in L304?

Just want to make sure the object here doesn't need to adhere to a stricter type.

Reply:
The definition of `LoadBalancingConfig` is actually pretty strict. You can see it here: https://github.com/grpc/grpc-node/pull/1517/files#diff-4d287ea20ecc388ea1b8d8fa6eea5af0R137

There are some config types that wouldn't work there, but the idea is that the object just describes a tree of load balancing policies, and the concrete load balancer objects at each level can decide whether they can function with the information they are given.

Reply:
Ah I see that `LoadBalancingConfig` type definition now. Thanks for the pointer. That makes sense.